contrib/demo/clusters/kind/createKindClusters.sh

```diff
@@ -15,11 +15,56 @@
 # limitations under the License.
 
 set -o errexit
+set -o nounset
+set -o pipefail
+set -o xtrace
 
-DEMO_ROOT="$(dirname "${BASH_SOURCE}")/../.."
+DEMO_ROOT="$(dirname "${BASH_SOURCE[0]}")/../.."
 
-kind create cluster --config ${DEMO_ROOT}/clusters/kind/us-east1.config --kubeconfig ${DEMO_ROOT}/clusters/kind/us-east1.kubeconfig
-sed -e 's/^/    /' ${DEMO_ROOT}/clusters/kind/us-east1.kubeconfig | cat ${DEMO_ROOT}/clusters/us-east1.yaml - > ${DEMO_ROOT}/clusters/kind/us-east1.yaml
+get_cluster_port() {
+    local cluster=kind-$1
+    local clusterPort
+    clusterPort=$(kubectl --kubeconfig "${CLUSTERS_DIR}/$1.kubeconfig" config view -o json | jq -r ".clusters[] | select(.name==\"${cluster}\").cluster.server" | cut -d ':' -f 3)
+    echo "${clusterPort}"
+}
 
-kind create cluster --config ${DEMO_ROOT}/clusters/kind/us-west1.config --kubeconfig ${DEMO_ROOT}/clusters/kind/us-west1.kubeconfig
-sed -e 's/^/    /' ${DEMO_ROOT}/clusters/kind/us-west1.kubeconfig | cat ${DEMO_ROOT}/clusters/us-west1.yaml - > ${DEMO_ROOT}/clusters/kind/us-west1.yaml
+podman_ssh() {
+    local podmanMachine
+    podmanMachine="${PODMAN_MACHINE:-podman-machine-default*}"
+
+    local sshURL
+    sshURL=$(podman system connection ls --format=json | jq -r ".[]|select(.Name==\"${podmanMachine}\").URI" | grep -E -o 'ssh://[^:]+:\d+')
```

Comment:
On my RHEL linux, `sshURL` is empty, obviously because `podman system connection ls` is also empty.
So the script stops here, and I cannot go further in the tests.

Do we miss some condition here, in order to fallback to the previous behavior ?

Reply:
Oh, so you have both podman & docker installed?

Reply:
Seems yes :-)

Reply:
K, will fix

Reply:
This works now.